src/SystemLoader.cc

```diff
@@ -63,16 +63,7 @@ class gz::sim::SystemLoaderPrivate
   public: bool InstantiateSystemPlugin(const sdf::Plugin &_sdfPlugin,
               gz::plugin::PluginPtr &_gzPlugin)
   {
-    // Deprecated: accept ignition-gazebo-prefixed systems
```

Comment:
We sort of discussed this in the weekly meeting. Removing this means any existing SDFormat file that has the `ignition-gazebo` plugins will not work in Harmonic. Are we okay with that?